src/handles/handle.rs

```diff
@@ -263,6 +266,14 @@ pub mod auth {
         AuthHandle
     );
     impl_handle_conversion!(AuthHandle, ObjectHandle);
+    // The following constant handles can be used for authorization
+    // according to the TCG TPM2 r1p59 Structures specification.
+    add_constant_handle!(AuthHandle, OwnerHandle, ESYS_TR_RH_OWNER);
+    add_constant_handle!(AuthHandle, Lockout, ESYS_TR_RH_LOCKOUT);
+    add_constant_handle!(AuthHandle, EndorsementHandle, ESYS_TR_RH_ENDORSEMENT);
+    add_constant_handle!(AuthHandle, PlatformHandle, ESYS_TR_RH_PLATFORM);
+    // TODO: Figure out how to add AUTH_00 to AUTH_FF range
+    // TODO: Figure out how to add ACT_0 to ACT_F range
```

Comment:
Which ones are these? I can see the auth ones ( `#define ESYS_TR_RH_AUTH(x)     (ESYS_TR_RH_AUTH_FIRST +(ESYS_TR)(x))` ) but I don't get what the ACT range is.

Reply:
It comes from the very part I explicitly did not wanted to cover in this PR. The permanent handles (**TCG TPM2 r1p59 Part 2 Structures: Chapter 7.4 Permanent Handles**). I chose to just cover the range and leave it at that. But in the future I think they need to be specified further.

Reply:
If you want to read more about the **Authenticated Countdown Timer** them self look at **TCG TPM2 r1p59 Part 3 Commands: Chapter 33 Authenticated Countdown Timer**.

Reply:
Cheers, that's what I was looking for, I'll have a read!